components/_host-profiles.scss

```diff
@@ -33,6 +44,36 @@ host profiles.
 	}
 }
 
+.social-container {
+	text-align: center;
+	padding-top: 0.5em;
+
+	.social {
+    display: inline-block;
+    padding: 0 .25em;
```

Comment:
`.25` should be written with a leading zero as `0.25`<br>Properties should be ordered display, font-size, font-weight, padding, text-transform